<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeSelectionDialog</class>
 <widget class="QDialog" name="TimeSelectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="doctorInfoLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCalendarWidget" name="calendarWidget"/>
     </item>
     <item>
      <widget class="QListWidget" name="timeSlotsListWidget"/>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="confirmButton">
     <property name="text">
      <string>确认预约</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
